### DISCUSSION TITLE
是否需要把Wiki链接换回BWiki

### DISCUSSION THREAD

**Discussion by idhaname** [Meloong-Git/PCL#3488] [提问] [answered]
[中文Minecraft Wiki在BWiki的镜像重新开放](https://www.bilibili.com/opus/887141687280795704)
因为用了知名的Cloudflare减速CDN，部分地区有时候[新站](zh.minecraft.wiki)的访问速度较慢
是否考虑把Wiki链接换到B站的镜像站，或者在设置里做一个选项让用户切换
（这个功能做起来肯定很蛋疼，就先不往Issues里发了）

**Comment by zkitefly** [comment, marked as answer]
https://github.com/Hex-Dragon/PCL2/issues/1537#issuecomment-1492867308
